### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.NullPointerException

### ISSUE THREAD

**Issue by matthiasguentert** [microsoft/azure-tools-for-java#6605]
IntelliJ build version: 2022.1 IU-221.5080.210
OS: Windows 11
JDK: JetBrains s.r.o. 11.0.14.1
Plugin version: 3.64.0-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,7,0,1693x921,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.NullPointerException
	at com.microsoft.intellij.util.MavenUtils.getRootMavenProject(MavenUtils.java:111)
	at com.microsoft.azure.toolkit.intellij.legacy.function.wizard.module.helper.MavenFunctionsModuleBuilderHelper.lambda$setupMavenModule$1(MavenFunctionsModuleBuilderHelper.java:47)
```

**Comment by wangmingliang-ms** [comment]
This bug would be fixed by commit d199625e, it will be included in the next stable release.

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
